tests/utils/TestHelper.js

```diff
@@ -5,18 +5,43 @@ import HttpUtils from '../../src/libs/HttpUtils';
 import ONYXKEYS from '../../src/ONYXKEYS';
 import waitForPromisesToResolve from './waitForPromisesToResolve';
 import * as ReportUtils from '../../src/libs/ReportUtils';
+import * as NumberUtils from '../../src/libs/NumberUtils';
+
+/**
+ * @param {String} login
+ * @param {Number} accountID
+ * @param {String} [firstName]
+ * @returns {Promise}
```

Comment:
We're returning an object instead of a promise here
```suggestion
 * @returns {Object}
```